identity/client/src/pages/users/detail/authorization/List.tsx

```diff
@@ -47,23 +112,150 @@ const List: FC<AuthorizationsListProps> = ({ user, loadingUser }) => {
+                        </TableExpandRow>
+                        {expandedRows[row.id] && (
+                          <TableExpandedRow colSpan={headers.length + 2}>
+                            {/* Displaying list of permissions */}
+                            {dataMap[row.id]?.permissions && (
+                              <ul>
+                                {dataMap[row.id].permissions.map(
+                                  (permission, index) => (
+                                    <React.Fragment key={index}>
+                                      <li
+                                        style={{
+                                          display: "flex",
+                                          justifyContent: "space-between",
+                                          alignItems: "center",
+                                          padding: "10px 0",
+                                        }}
+                                      >
+                                        <div>
+                                          <strong>Permission Type:</strong>{" "}
+                                          {permission.permissionType}
+                                          <br />
+                                          <strong>Resource IDs:</strong>{" "}
+                                          {permission.resourceIds.join(", ")}
+                                        </div>
+                                        {/* Delete button at the end */}
+                                        <button
+                                          type="button"
+                                          onClick={() =>
+                                            onEditPermission(row.id, index)
+                                          }
+                                          style={{
+                                            border: "none",
+                                            background: "none",
+                                            cursor: "pointer",
+                                          }}
+                                          aria-label="Delete permission"
+                                        >
+                                          <Edit />
+                                        </button>
+                                      </li>
+                                      {/* Divider between permission rows */}
+                                      {index <
+                                        dataMap[row.id].permissions.length -
+                                          1 && (
+                                        <hr
+                                          style={{
+                                            border: "none",
+                                            borderTop: "1px solid #ccc",
+                                            margin: "5px 0",
+                                          }}
+                                        />
```

Comment:
I believe this can be done with CSS.
We also should use the Carbon spacing variables and color variables